integration/images.go

```diff
@@ -166,10 +173,10 @@ func (d *DockerFileBuilder) BuildImage(imageRepo, gcsBucket, dockerfilesPath, do
 
 	contextFlag := "-c"
 	contextPath := buildContextPath
-	for _, d := range bucketContextTests {
+	for _, d := range contextTest {
```

Comment:
Instead of changing this, we should have this logic twice (once for the bucket tests and once for the git tests). 

It could look like this:

```golang
for _, d := range bucketContextTests {
  if d == dockerfile {
    contextFlag = "-b"
   contextPath = gcsBucket
 }
}

for _, d := range gitRepoTests {
  if d == dockerfile {
     contextPath = "git://https://github.com/GoogleContainerTools/kaniko"
  }

}

```